bitnami/mongodb-sharded/README.md

```diff
@@ -87,6 +87,7 @@ The command removes all the Kubernetes components associated with the chart and
 | `usePasswordFile`                     | Mount credentials as files instead of using environment variables                                                                                         | `false`                   |
 | `shards`                              | Number of shards to be created                                                                                                                            | `2`                       |
 | `common.useHostnames`                 | Enable DNS hostnames in the replica set config                                                                                                            | `true`                    |
+| `common.mongodbEnableNumactl`         | Enable "numactl --interleave=all" before mongd instances                                                                                                  | `false`                    |
```

Comment:
Small typo in `mongd`.
Additionally, please use the same description as in the `values.yaml`:
```
Enable launch MongoDB instance prefixed with "numactl --interleave=all"
```